docs/FAQ.md

```diff
@@ -68,6 +68,18 @@ Your can find `myStorageAccount` to your registry by the following command
 az acr show -n myRegistry --query storageAccount
 ```
 
+## How to delete all manifests that are not referenced by any tag in a repository?
+
+If you are on bash
+```
+az acr repository show-manifests -n myRegistry --repository myRepository --query "[?tags==null].digest" -o tsv  | xargs -I% az acr repository delete -n myRegistry -t myRepository@%
+```
+
+For Powershell
+```
+az acr repository show-manifests -n myRegistry --repository myRepository --query "[?tags==null].digest" -o tsv | %{ az acr repository delete -n myRegistry -t myRepository@$_ }
```

Comment:
NIT: add a comment to use `-y` to skip confirmation.

Reply:
Added

Reply:
Added